### PR TITLE
Styled artsit

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by LauraCourquin — DISMISSED** [review]
It's good for me

**Review by LauraCourquin — DISMISSED** [review]
It's good for me !

**Review by LuigiMenez — DISMISSED** [review]
Ok for me. But I would have put the "/" Home route first

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -6,15 +6,15 @@ import Home from "@pages/Home";
...
-  </div>
+    <div className="App">
+      <Routes>
+        <Route path="/search" element={<Search />} />
+        <Route path="/" element={<Home />} />      
+        <Route path="/profile" element={<Profile />} />
+        <Route path="/profile/:profileId" element={<Profile />} />
+      </Routes>
+      <Navbar />
+    </div>
```
Comment (LoicBrassart) [inline frontend/src/App.jsx:18]: Why is this file touches at all ?
On the plus side, you seem to fix indentation
On the minus side, you create a useless empty space...
Prettier didn't go through this, did it ?

**Review by LoicBrassart — DISMISSED** [review]
Stop editing files on GHub -_-

**Review by LauraCourquin — APPROVED** [review]
It's good for me !

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request closed without merging**